frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -341,6 +348,29 @@ class ChartSettings extends Component {
             </div>
           </div>
         )}
+        <ChartSettingsWidgetPopover
```

Comment:
Yeah, that's on my list of things to fix up

Reply:
I try taking another stab at this. Seems pretty impossible to fix this in a short span of time. What do you think about modifying the popup so it says something like `no settings` or something, so it doesn't look broken? @cdeweyx 

p.s. I'm aware that this happens currently, so it's not a regression.

Reply:
Yeah I'm happy with displaying `Nothing to see here` in the short term. We can revisit a fix later on but let's not drop everything and refactor for this case since it's definintely not trivial.